src/components/Modals/ConfirmTransaction/components/index.ts

```diff
@@ -0,0 +1,184 @@
+import styled, { css } from 'styled-components/macro';
+import { colors, GU } from 'components/theme';
+import { media } from 'components/breakpoints';
+
+export const StyledCloseContainer = styled.button`
+  align-items: center;
+  background: transparent;
+  border: none;
+  display: flex;
+  fill: ${colors.white};
+  justify-content: center;
+  position: absolute;
+  right: ${1 * GU}px;
+  top: ${2 * GU}px;
+  transition: all 0.3s ease;
+
+  &:hover,
+  &:active,
+  &:focus {
+    cursor: pointer;
+    fill: ${colors.yellow};
+  }
+`;
+
+export const StyledCoinContainer = styled.div`
+  height: ${12 * GU}px;
+  width: ${12 * GU}px;
+  margin-right: ${4 * GU}px;
+`;
+
+interface IStyledContentContainer {
+  copied?: boolean;
+}
+
+export const StyledContentContainer = styled.div<IStyledContentContainer>`
+  border-bottom: ${GU}px solid #000000;
+  box-sizing: border-box;
+  padding: ${4 * GU}px;
+
+  button {
+    background: transparent;
+    border: none;
+
+    ${(props) =>
+      props.copied &&
+      css`
+        background: #000;
+        border-radius: 5px;
```

Comment:
We can make this `${GU / 2}` to stick with the scale. 😄

Reply:
Fixed in 71975a4375568e3aabea19bccf7b2865a5342264